language-support/scala/codegen-sample-app/BUILD.bazel

```diff
@@ -52,6 +52,7 @@ da_scala_library(
     scalacopts = [
         "-Wconf:cat=unused-imports&site=com\\.daml\\.sample\\..*:s",
         "-Wconf:msg=parameter value ev.. in method ContractIdNT (Value)|(LfEncodable) is never used:s",
+        "-Wconf:msg=Wrap `enum` in backticks to use it as an identifier:s",
```

Comment:
Do you still need this?

Reply:
Yes because the generated scala code contains `enum` as variable name :sweat:

Reply:
^ @stefanobaghino-da you might want to fix that in the scala codegen.

Reply:
If Stefano agrees to it I can take care of that too. However I wanted to keep the scope of this PR as much limited as possible so I would do it in a different PR.

Reply:
> ^ @stefanobaghino-da you might want to fix that in the scala codegen.

I'll add a ticket to track this, I don't think we should be working on this right now.

Reply:
https://github.com/digital-asset/daml/issues/12643